phoenix-core/src/main/java/org/apache/phoenix/expression/AndOrExpression.java

```diff
@@ -36,6 +36,8 @@
 public abstract class AndOrExpression extends BaseCompoundExpression {
     // Remember evaluation of child expression for partial evaluation
     private BitSet partialEvalState;
+    // true if we have seen NULL as the value of some child expression
```

Comment:
Redundant comment. Please remove.

Reply:
IMO this comment makes it easier to understand the code.